### DISCUSSION TITLE
empty config file

### DISCUSSION THREAD

**Discussion by Levan88** [entropyxyz/community#16] [Support]
I closed the terimanal window with the signature file open, the config file cleared, and I was unable to run yarn start (error below) and lost my wallet. 
```
/root/entropy/lev/cli/src/config/index.ts:21
  return JSON.parse(configBuffer.toString())
              ^
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.get (/root/entropy/lev/cli/src/config/index.ts:21:15)
    at async main (/root/entropy/lev/cli/src/index.ts:67:22)
error Command failed with exit code 1.
```

**Comment by johnnymatthews** [comment]
Well that's not ideal. When you say:

> with the signature file open

What do you mean? Did you have the `~/.entropy-cli.config` open in a text editor? Or were you within the **Sign** function within the CLI?

**Reply by johnnymatthews** [reply]
Haven't gotten a response. Gonna close this as inactive.
